docs/langref/src/concepts/file.md

```diff
@@ -37,10 +38,16 @@ export component MyApp inherits Window {
 
 ```
 
-Here, both `MyButton` and `MyApp` are components. `Window` and `Rectangle` are built-in elements
-used by `MyApp`. `MyApp` also re-uses the `MyButton` component.
+Both `MyButton` and `MyApp` are components. `Window` and `Rectangle` are built-in elements
+used by `MyApp`. `MyApp` also re-uses the `MyButton` component as two separate elements.
+
+Elements have properties, which you can assign values to. Here the first `MyButton`
+element has a property `text` that we assign a string constant "hello" to. You
+can assign expressions using other properties as well. Slint will re-evaluate
+properties when any of the properties they depend on change, which makes the
+user-interface reactive.
```

Comment:
`MyBytton` has a `text` property only through its inheritance, so you could argue that `Text` is the one that has the property. I'd omit the "ownership" part here, to avoid confusion, and go straight to the assignment. We've already established that every element has properties after all :)

```suggestion
Elements have properties, which you can assign values to. Here we assign a string
constant "hello" to the first `MyButton`'s `text` property. You
can also assign entire expressions. Slint will re-evaluate the expressions when any
of the properties they depend on change, which makes the user-interface reactive.
```